### ISSUE TITLE
Fetaure Enbales/Disable AstroPrint

### ISSUE THREAD

**Issue by Antebur** [AstroPrint/OctoPrint-AstroPrint#89]
it would be nice to have a function to enable/disable Astroprint from Octoprint plugin while not in use without to delete Astroprint account.

**Comment by Alfonsopl** [comment]
I do not really understand the feature request .
You already can disable it by Octoprint Settings -> plugin manager.

<img width="1010" alt="Captura de pantalla 2020-04-02 a las 19 56 07" src="https://user-images.githubusercontent.com/14834229/78282348-6c04a800-751c-11ea-8c3a-79179ecf6d5b.png">

If you disable it, and you enable it within 14 days, you will still be connected in your account.

**Comment by CoDanny** [comment]
@Alfonsopl What I think it's being requested here is the ability to unregister from the cloud  ( and stay disconnected ) without logging the user out and then the ability to connect to register to cloud again without having to enter go through the auth flow.

**Comment by Antebur** [comment]
yes true on the Astroprint Tab of octoprint the abiltity to register/unregstier
